### PR TITLE
New PR for margins in stable - fixes  #9651

### LINKED ISSUE #9651: Pets and Mounts in the Stable need margin updates
Should have ~24px between each Pet/Mount.

**Current**
image

**Designed**
image

If you would like to work on this issue, please comment an email address to be invited to our Zeplin project.

### PR DESCRIPTION
**Pull request by seekheart**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
new pr for #9903 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
see #9903 screenshots 

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID: 0848ed18-0966-4d90-83da-a1b4ae1500cc

Fixes #9651

### CODE REVIEW

File: website/client/components/inventory/item.vue [hunk 1]
```diff
@@ -22,6 +22,9 @@ div(v-else)
     slot(name="popoverContent", :item="item")
 </template>
 
+<style>
```
Comment (negue) [inline website/client/components/inventory/item.vue]: could you remove the empty style tag?

File: website/client/components/inventory/stable/petItem.vue [hunk 2]
```diff
@@ -3,7 +3,7 @@ div
   .item-wrapper(@click="click()", :id="itemId")
     .item(
       :class="{'item-empty': emptyItem, 'highlight': highlightBorder}",
-    )
+    ).pet-slot
```
Comment (negue) [inline website/client/components/inventory/stable/petItem.vue]: `.pet-slot` should be after `.item` before the attributes

**Review by negue — CHANGES_REQUESTED** [review]
Thanks for your PR 👍 

Could you please fix the mentioned fixes and revert the changes on:
- [ ] website/static/presskit/Habitica - Gamify Your Life.mp4
- [ ] website/static/presskit/presskit.zip

**Comment by seekheart** [comment]
@negue how do I revert the changes on those files?

File: website/client/components/inventory/stable/petItem.vue (hunk 2 above)
Comment (seekheart) [reply website/client/components/inventory/stable/petItem.vue]: it already is?

File: website/client/components/inventory/item.vue (hunk 1 above)
Comment (seekheart) [reply website/client/components/inventory/item.vue]: removed

File: website/client/components/inventory/stable/petItem.vue (hunk 2 above)
Comment (seekheart) [reply website/client/components/inventory/stable/petItem.vue]: can you give me a snippet for clarity please?

Comment (negue) [reply website/client/components/inventory/stable/petItem.vue]: sorry what I meant was `.item.pet-slot(`

Comment (seekheart) [reply website/client/components/inventory/stable/petItem.vue]: alrighty, I couldn't `git revert` the two files you requested, is there an easier way?

Comment (seekheart) [reply website/client/components/inventory/stable/petItem.vue]: also for the stuff under `.pet-slot` lines 7 I left them be.

**Comment by negue** [comment]
to revert you can use this 

`git checkout origin/develop -- PATH/FILE`  for both mentioned files and then just commit, then those files will have the same content again ^^

**Comment by seekheart** [comment]
does that still apply if I forked the repo?

**Comment by negue** [comment]
yes it should work too, since `HabitRPG/habitica` should've been added as remote named `origin` and `origin/develop` points to our `develop` branch

**Comment by seekheart** [comment]
according to the wiki it was named `upstream`? so still `origin` or `upstream`?

**Comment by negue** [comment]
could you check how your remotes are named? mine still is at origin , maybe I added it myself

**Comment by seekheart** [comment]
it's upstream for me, I can't push the changes in till tmr anywho. I think I nailed everything you wanted. :)

**Comment by negue** [comment]
ok thank you 👍

**Comment by seekheart** [comment]
sorry to be a bother @negue I think I just successfully pushed the changes here :)

**Comment by negue** [comment]
@seekheart  yes but now we also need to revert the `saving pngs` commit, I'd suggest to always use a different branch for fixes  / features, not to push it always to `develop` :)

**Comment by seekheart** [comment]
all set, thanks for the advice @negue hopefully this is the last thing I need to push xD.... in the future I'll use a feature branch

**Comment by negue** [comment]
it seems the files are still changed :/ maybe its just easier to create a third PR with only that css change and nothing else. and make you sure branch from `upstream/develop` not `develop`(of your local branch) that way you always have the latest changes

**Comment by Alys** [comment]
@seekheart Can you remove the mp4 and zip file too? They're likely to have been modified in your local install due to a problem that's unrelated to your changes, but you can avoid having irrelevant files included in a PR by committing only the files you changed. E.g., `git commit filea fileb` rather than `git commit -a`.

If you're not sure how to remove them, the info at the first couple of links [from this search](https://www.google.com.au/search?hl=en&lr=&btnG=Search&tbo=1&q=remove+files+from+github+pull+request&gws_rd=cr&dcr=0&ei=_lZ_WuamAYH28AW42p5I) should help.

**Comment by seekheart** [comment]
I think I got it now @Alys the linked help and the two files are gone. @negue only css changes now :D

File: website/client/components/inventory/stable/petItem.vue [hunk 3]
```diff
@@ -28,10 +28,18 @@ div
...
 
+  .pet-slot {
+    margin: 0 2px;
+  }
+
   .pet-progress-bar {
     height: 4px;
     background-color: #24cc8f;
   }
+
```
Comment (negue) [inline website/client/components/inventory/stable/petItem.vue]: could you please remove the duplicate css rule?

Comment (seekheart) [reply website/client/components/inventory/stable/petItem.vue]: duplicate css rule? @negue

Comment (seekheart) [reply website/client/components/inventory/stable/petItem.vue]: nvm I took care of it ;) @negue

**Comment by Alys** [comment]
@seekheart Can you also please do the Mounts? The issue mentions them as well and both should be modified together so it doesn't look inconsistent.

**Comment by seekheart** [comment]
@Alys are CSS classes global if I don't have `<style scope>` declaration? I figured I can just add `.pet-slot` to `mountItem.vue`?

**Comment by paglias** [comment]
Yes, anything not in a `<style scoped>` slot is global, but in that case for global css it's better to use the assets/scss folder

**Comment by seekheart** [comment]
@Alys @negue pushed the changes to mounts as well.

File: website/client/components/inventory/stable/mountItem.vue [hunk 4]
```diff
@@ -15,6 +15,11 @@ div
     slot(name="popoverContent", :item="item")
 </template>
 
+<style lang="scss">
```
Comment (paglias) [inline website/client/components/inventory/stable/mountItem.vue]: since this style block is not scoped can you move it to the assets/scss folder?

Comment (paglias) [reply website/client/components/inventory/stable/mountItem.vue]: It's also a duplicate of the other change you made so you should just keep one version and move it to assets/scss

**Review by paglias — CHANGES_REQUESTED** [review]

File: website/client/components/inventory/stable/mountItem.vue (hunk 4 above)
Comment (seekheart) [reply website/client/components/inventory/stable/mountItem.vue]: when I move to assets/scss is that going to be a new file and if so how do I call it? Is it called automatically from some build task?

Comment (paglias) [reply website/client/components/inventory/stable/mountItem.vue]: I would put it into the existing item.scss file

**Comment by seekheart** [comment]
@paglias @negue it's now in one place under `item.scss` in assets.

File: website/client/assets/scss/item.scss [hunk 5]
```diff
@@ -19,6 +19,10 @@
   margin-bottom: 8px;
 }
 
+.pet-slot {
```
Comment (paglias) [inline website/client/assets/scss/item.scss]: just one small change and this should be good to go, can you change the selector from `.pet-slot` to `.item.pet-slot` so it's more specific?

**Review by paglias — CHANGES_REQUESTED** [review]

File: website/client/assets/scss/item.scss (hunk 5 above)
Comment (seekheart) [reply website/client/assets/scss/item.scss]: done

**Comment by paglias** [comment]
@seekheart  thanks for this! I'm merging the PR and awarding your contributor tier.

Note that further tiers require increasing amounts of work from one to the next, so Tier 2 may require a couple of PRs or a larger PR to attain. But keep helping out and we'll express our gratitude accordingly!

**Comment by seekheart** [comment]
@paglias it was fun contributing :)

**Comment by paglias** [comment]
@seekheart great! Sorry for the time it took to merge this in :)

**Pull request merged**